### PR TITLE
(79) Problem description page (known)

### PR DESCRIPTION
**Pull request by leeky**
Create a dummy form for entering details about the repair.

NB: Anything saved into the form is currently **NOT** persisted. Additionally, submitting the form redirects the user to the address search page.

As part of this branch, I've included the GOVUK focus styles (which highlight the currently-focussed form element) and broken the stylesheet up into smaller parts.

From: `/questions/describe-repair` 

describe-repair

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -6,6 +6,9 @@
   namespace :questions do
     get '/start', to: 'start#index', as: 'start'
     post '/start', to: 'start#submit', as: 'start_submit'
+
+    get '/describe-repair', to: 'describe_repair#index', as: 'describe_repair'
+    post '/describe-repair', to: 'describe_repair#submit'
```
Comment (dgmstuart) [inline config/routes.rb]: Does this belong under the 'questions' path, or is it its own thing? Maybe it doesn't matter at this point - easy to change later?

File: app/controllers/questions/describe_repair_controller.rb [hunk 2]
```diff
@@ -0,0 +1,10 @@
+module Questions
+  class DescribeRepairController < ApplicationController
+    def index; end
+
+    def submit
+      # redirect_to 'next_form'
```
Comment (dgmstuart) [inline app/controllers/questions/describe_repair_controller.rb]: Should this redirect to the address search page?

**Review by dgmstuart — CHANGES_REQUESTED** [review]
Looks good - I think we can join this up to the next page?

Stylesheet cleanup is nice 👍

File: config/routes.rb (hunk 1 above)
Comment (leeky) [reply config/routes.rb]: Possibly. As this is a named route it'd be easy to change in the future.

Comment (leeky) [reply config/routes.rb]: On second thoughts, I agree with you. I'll move these out of the 'questions' namespace.

**Review by dgmstuart — APPROVED** [review]

**Comment by dgmstuart** [comment]
LGTM 👍

**Pull request merged**